compiler/Eta/Backpack/NameShape.hs

```diff
@@ -150,7 +150,7 @@ ns_module = mkHoleModule . ns_mod_name
 -- | Substitution on @{A.T}@.  We enforce the invariant that the
 -- 'nameModule' of keys of this map have 'moduleUnitId' @hole@
 -- (meaning that if we have a hole substitution, the keys of the map
--- are never affected.)  Alternately, this is ismorphic to
+-- are never affected.)  Alternately, this is isomorphic to
```

Comment:
Most corrections were automatically suggested by Google Sheets. This is amongst them. In general, I don't fight Google except when I have sufficient experience to know it's wrong.